src/ngraph/op/convert.cpp

```diff
@@ -55,3 +56,38 @@ void op::Convert::generate_adjoints(autodiff::Adjoints& adjoints, const OutputVe
 
     adjoints.add_delta(x, make_shared<op::Convert>(delta, x.get_element_type()));
 }
+
+namespace
+{
+    template <element::Type_t ET>
+    bool try_evaluate_convert(const EvaluatorTensorPtr& arg,
+                              const EvaluatorTensorPtr& out,
+                              size_t count)
```

Comment:
You need a separate expansion of element types for the `out` argument types, i.e. `try_evaluate_convert` needs to do the same thing `evaluate_convert` does, but for the output type instead of the input type.

Reply:
done